crates/ruma-events/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ The new format (Session) is required to reliably display the call member count (
 `CallMemberEventContent` is now an enum to model the two different formats.
 - `CallMemberStateKey` (instead of `OwnedUserId`) is now used as the state key type for `CallMemberEventContent`.
 This guarantees correct formatting of the event key.
+- Add support for captions on audio, file, image and video messages.
```

Comment:
I would not say "add support", captions are already supported. Maybe "add helpers"?